src/Codebase/Infrastructure/ProjectConfigurationParser/ProjectConfigurationParser.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Codebase\Infrastructure\ProjectConfigurationParser;
+
+use Codebase\Application\Dto\ConfigurationRequestDto;
+use Codebase\Application\Dto\ConfigurationResponseDto;
+use Codebase\Infrastructure\Exception\ProjectConfigurationFileInvalidSyntaxException;
+use Exception;
+use Symfony\Component\Yaml\Yaml;
+
+class ProjectConfigurationParser implements ProjectConfigurationParserInterface
+{
+    /**
+     * @var string
+     */
+    public const UPGRADER_KEY = 'upgrader';
+
+    /**
+     * @var string
+     */
+    public const PREFIXES_KEY = 'prefixes';
+
+    /**
+     * @var string
+     */
+    public const DEFAULT_PREFIX = 'Pyz';
+
+    /**
+     * @var string
+     */
+    public const INVALID_TYPE_ERROR_MESSAGE = 'Value of %s.%s should be array of string';
```

Comment:
Please don't use constants for the exception messages as they are not reusable.

Reply:
Thanks ++